test.py

```diff
@@ -115,6 +116,16 @@ def test_validate_fast(self):
                     "2478433644_2839c5e8b8_o_d.jpg"))
         self.assertRaises(bagit.BagValidationError, self.validate, bag, fast=True)
 
+    def test_validate_completeness(self):
```

Comment:
The idea was just that the body of that test could patch the `_validate_entries` method and confirm that it's not accidentally being called. If I remember the API correctly, that's something like this:

```python

with mock.patch.object(bag, '_validate_entries') as m:
    self.assertTrue(self.validate(bag, fast=True))
    self.assertEqual(m.call_count, 0)
```